clients/da-vinci-client/src/main/java/com/linkedin/davinci/consumer/VeniceChangelogConsumer.java

```diff
@@ -198,12 +198,11 @@ public interface VeniceChangelogConsumer<K, V> {
   Collection<PubSubMessage<K, ChangeEvent<V>, VeniceChangeCoordinate>> poll(long timeoutInMs);
 
   /**
-   * Checks whether all subscribed partitions are caught up during bootstrap.
-   * @param offsetThresholdTimestamp Offset timestamp in millisecond. If a partition's (startTimestamp - heartbeatTimestamp)
-   *                                 is smaller or equal to offsetThresholdTimestamp, we consider this partition is caught up.
+   * Checks whether all subscribed partitions are caught up during bootstrap. If a partition's (startTimestamp - heartbeatTimestamp)
```

Comment:
As I tried to say in my last comment, relying on the `startTimestamp` is confusing... I think the formula should be:

```
current timestamp - timestamp of the last consumed message < 1 minute
```

Reply:
Our original discussion was concerning about chasing moving target for current TS with data message in less frequently updated partition will lead to slowness in catchup.

But I guess with 1 min fixed constant I guess it is all fine, I can adjust that.

Reply:
If what we actually want to build is to measure if we caught up to the start time (or to some arbitrary time), then I think we cannot call this API `isCaughtUp`, because the meaning is too different, IMHO...